src/services/WolfDen.Application/DTOs/Employees/EmployeeDirecotyWithPageCountDTO.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WolfDen.Application.DTOs.Employees
+{
+    public class EmployeeDirecotyWithPageCountDTO
```

Comment:
Rename Class and file name

Reply:
renamed
how to use this wrapper class as common wrapper class since its attributes are pagecount and the dto list?